src/ReportCommand.php

```diff
@@ -141,6 +174,25 @@ protected function outputFormattedLicenses(OutputInterface $output, array $licen
                 ]);
             }
             $licenseTable->render();
+
+            if ($input->getOption('show-packages') || $output->isVerbose()) {
+                $output->writeln('');
+
+                if ($input->getOption('grouped')) {
+                    $output->write('packages: ');
+                    $packages = [];
+                    foreach ($dependencies as $dependency) {
+                        $packages[] = $dependency->getName();
+                    }
+
+                    $output->write(implode(', ', $packages));
+                    continue;
```

Comment:
Deeply nested `continue` statements make the code hard to read. Try to avoid that.